client/package.json

```diff
@@ -13,13 +20,8 @@
     "react-dom": "^17.0.1",
     "react-router-dom": "^6.2.1",
     "react-scripts": "4.0.1",
-    "web-vitals": "^0.2.4",
```

Comment:
why are these dependencies removed?

Reply:
i dont know why...i didn't edit the package.json file

Reply:
they are not deleted..i added the following dependencies, the location of the other dependencies moved
"@fortawesome/free-solid-svg-icons": "^6.2.0",
 "@fortawesome/react-fontawesome": "^0.2.0",
 
You can see all the dependencies when you view the whole file

Reply:
ooh okay, my bad